src/libraries/System.Net.NetworkInformation/src/System/Net/NetworkInformation/MulticastIPAddressInformation.cs

```diff
@@ -11,31 +13,37 @@ public abstract class MulticastIPAddressInformation : IPAddressInformation
         /// <summary>
         /// Gets the number of seconds remaining during which this address is the preferred address.
         /// </summary>
+        [SupportedOSPlatform("windows")]
         public abstract long AddressPreferredLifetime { get; }
 
         /// <summary>
         /// Gets the number of seconds remaining during which this address is valid.
         /// </summary>
+        [SupportedOSPlatform("windows")]
         public abstract long AddressValidLifetime { get; }
 
         /// <summary>
         /// Specifies the amount of time remaining on the Dynamic Host Configuration Protocol (DHCP) lease for this IP address.
         /// </summary>
+        [SupportedOSPlatform("windows")]
         public abstract long DhcpLeaseLifetime { get; }
 
         /// <summary>
         /// Gets a value that indicates the state of the duplicate address detection algorithm.
         /// </summary>
+        [SupportedOSPlatform("windows")]
         public abstract DuplicateAddressDetectionState DuplicateAddressDetectionState { get; }
 
         /// <summary>
         /// Gets a value that identifies the source of a unicast IP address prefix.
         /// </summary>
+        [SupportedOSPlatform("windows")]
         public abstract PrefixOrigin PrefixOrigin { get; }
 
         /// <summary>
         /// Gets a value that identifies the source of a unicast IP address suffix.
         /// </summary>
+        [SupportedOSPlatform("windows")]
```

Comment:
All APIs annotated, could be moved to the type

Reply:
How does it behave then for the properties on the parent type? The parent type contains properties which are supported on other platforms.

Reply:
For now there is no analysis for the attributes applied to parent/child (though as per https://github.com/dotnet/runtime/issues/72513#issuecomment-1191738659 it might be added in the future)  

Anyway,  you can ignore this comment as you might want to add an API to this type that works on other platforms